test/plugin_test.dart

```diff
@@ -89,7 +97,7 @@ void main() {
             '$cli $args debug-files upload $orgAndProject --include-sources $buildDir/app/outputs',
             '$cli $args releases $orgAndProject new $release',
             '$cli sourcemaps inject $buildDir/web/file.js $args $orgAndProject',
-            '$cli sourcemaps upload $buildDir/web --ext js --ext map ./ --ext dart $args $orgAndProject',
+            '$cli sourcemaps upload $buildDir/web --ext js --ext map --strip-prefix ../../../../ --strip-prefix ../../Documents --strip-prefix ../../ ./ --ext dart $args $orgAndProject',
```

Comment:
How are overlapping prefixes handled? Would running `../../` before `../../Documents` miss the latter, as there would be `/Documents` left? Is this something we'd need to handle on the client by calling the prefixes in correct order?

Reply:
added sorting